Microsoft.Toolkit.Services/Services/MicrosoftGraph/MicrosoftGraphAuthenticationHelper.cs

```diff
@@ -192,7 +211,7 @@ internal bool LogoutV2()
         /// <param name="resourceId">Azure AD application resource ID</param>
         /// <param name="promptBehavior">Prompt behavior</param>
         /// <returns>An oauth2 access token.</returns>
-        internal async Task<string> GetUserTokenAsync(string appClientId, string resourceId = MicrosoftGraphResource, PromptBehavior promptBehavior = PromptBehavior.Always)
+        public async Task<string> GetUserTokenAsync(string appClientId, string resourceId = MicrosoftGraphResource, PromptBehavior promptBehavior = PromptBehavior.Always)
```

Comment:
This Prompt behavior is bad. It should be `SelectAccount` by default so that it can leverage Connected To Windows accounts.

https://docs.microsoft.com/en-us/dotnet/api/microsoft.identitymodel.clients.activedirectory.promptbehavior?view=azure-dotnet

Reply:
Just created an issue for this to fix in a different PR. #2350